src/main/java/xml/XmlReader.java

```diff
@@ -94,18 +120,41 @@ public Document read(InputStream stream, String parentDir)
 	/**
 	 * Creates a DataFile from a file Element.
 	 * @return a new DataFile
+	 * @param elem The file element read from the xml file
+	 * @param parentDir The parent of the file
 	 */
 	public DataFile createDataFile(Element elem, String parentDir) {
```

Comment:
Nee er wordt hier geen data uit datafiles gehaald. Hier gebeurt alleen het specificeren van de datafile op basis van de geparste elementen van de xml en wordt er een datafile geproduceerd. Het echt lezen gebeurt pas als je in een datafile de getDataStream() methode aanroept